utils/strings.py

```diff
@@ -0,0 +1,31 @@
+# Python Standard Library Imports
```

Comment:
Please move these to https://github.com/hacktoolkit/django-htk/blob/master/utils/text/transformers.py or whichever file is most appropriate in that directory.